src/pybind/mgr/dashboard/frontend/e2e/rgw/buckets.e2e-spec.ts

```diff
@@ -1,11 +1,11 @@
 import { Helper } from '../helper.po';
-import { BucketsPage } from './buckets.po';
+import { PageHelper } from '../page-helper.po';
```

Comment:
The recommendation for storing the "new Helper().buckets" into a variable is valid and we will make that change when calling "navigateTo". However, for functions such as getBreadcrumbText (which are marked as static) we would have to change every single file that uses these functions (every e2e). This might be a bit beyond the scope of this PR, but we'd be more than happy to go back and change this!

Reply:
@tspmelo I like the idea to simplify the access in a test to these objects for all E2E tests.
@rafaelquint Adapting all ocurrences of this in all tests is definitely out of scope for this PR.
@callithea If you're fine with the proposal, I'll create a tracker ticket to keep track of this proposal.

Reply:
@tspmelo I like the proposal. @p-na +1 for creating a tracker issue, thanks!